frontend/src/components/common/header/header.module.scss

```diff
@@ -0,0 +1,113 @@
+a {
+  color: #000;
+}
```

Comment:
Don't use global styles in the component's styles. It should contain only classes